CONTRIBUTING.md

```diff
@@ -34,6 +34,8 @@ We use GitHub issues to track public bugs. We will keep a close eye on this and
 ### Reporting New Issues
 Not all issues are SDK issues. If you're unsure whether your bug is with the SDK or backend, you can test to see if it reproduces with our [REST API][rest-api] and [Parse API Console][parse-api-console]. If it does, you can report backend bugs [here][bug-reports].
 
+To view the REST API network requests issued by the Parse SDK, please check out our [Network Debugging Tool][network-debugging-tool].  With this tool, you can either log network requests to Android logcat, or log them to Chrome Debugger via Stetho.
```

Comment:
maybe add and the responses from Parse Server after To view the REST API network requests issued by the Parse SDK?